<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetSearch</class>
 <widget class="QWidget" name="WidgetSearch">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>724</width>
    <height>488</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_4">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayoutSearch">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,10,1">
         <item>
          <widget class="QComboBox" name="comboBoxDictionaryType">
           <item>
            <property name="text">
             <string>Eng - Eng</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Eng - Viet</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Viet - Eng</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Slang</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Emoji</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditSearch"/>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonSearch">
           <property name="text">
            <string>Search</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayoutSearchType">
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QRadioButton" name="radioButtonSearchKeyword">
           <property name="text">
            <string>Search by keyword</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioButtonSearchDictionary">
           <property name="text">
            <string>Search by definition</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="2,3">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QListWidget" name="listWidgetHistory"/>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QPushButton" name="pushButtonAddNewWord">
             <property name="text">
              <string>Add New Word</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtonRandomWord">
             <property name="text">
              <string>Random Word</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTabWidget" name="tabWidgetDefinition">
         <widget class="QWidget" name="tab">
          <attribute name="title">
           <string>Tab 1</string>
          </attribute>
         </widget>
         <widget class="QWidget" name="tab_2">
          <attribute name="title">
           <string>Tab 2</string>
          </attribute>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
